webanno-ui-annotation/src/main/java/de/tudarmstadt/ukp/clarin/webanno/ui/annotation/AnnotationPage.java

```diff
@@ -313,7 +313,7 @@ protected void onConfigure()
 
         add(new LambdaAjaxLink("showOpenDocumentModal", this::actionShowOpenDocumentDialog));
         
-        add(new LambdaAjaxLink("showPreviousDocument", t -> actionShowPreviousDocument(t))
+        add(new LambdaAjaxLink("showPreviousDocument", this::actionShowPreviousDocument)
```

Comment:
Commit 002614a message : Work around https://bugs.openjdk.java.net/browse/JDK-8141508 by using explicit lambdas when calling actions defined in AnnotationPageBase